src/KeylightUtilBundle/Resources/config/asset.yml

```diff
@@ -1,6 +1,8 @@
 services:
     keylight_util_asset_manager:
         class: KeylightUtilBundle\Services\Asset\AssetManager
+    KeylightUtilBundle\Services\Asset\AssetManager: '@keylight_util_asset_manager'
```

Comment:
would be nicer to just cleanup and get rid of the old definitions. We anyway use specific versions in all the projects. Therefore it shouldn't break anything